django/contrib/gis/db/models/fields.py

```diff
@@ -280,6 +255,15 @@ def formfield(self, **kwargs):
             defaults['widget'] = forms.Textarea
         return super().formfield(**defaults)
 
+    def select_format(self, compiler, sql, params):
+        """
+        Return the selection format string, depending on the requirements
+        of the spatial backend.  For example, Oracle and MySQL require custom
+        selection formats in order to retrieve geometries in OGC WKB.
+        """
+        select = compiler.connection.ops.select
```

Comment:
I think it doesn't worth it as increase complexity and doesn't reduce amount of code.